app/assets/stylesheets/_steps.scss

```diff
@@ -7,10 +7,10 @@ $light-grey-2: #f8f8f8;
 $black: #0B0C0C;
 $red: #b10e1e;
 $grey: #eeeeee;
-$dark-grey-2: #6f777b;
 $beige: #8a6d3b;
 $border-grey: #aaaaaa;
 $dark-grey: #999999;
+$dark-grey-2: #6f777b;
```

Comment:
Can you discard this change.

Reply:
this is being used at https://github.com/alphagov/collections-publisher/pull/674/files#diff-4db2a3f0387faabcad6f57ec14a0f689R232

Reply:
Sorry, I think the rebase has confused things, but this comment referred to when this line was moved down a few lines, which was an unnecessary change.

Reply:
I think commit (0d128fbe8f060b05d245ddbec9a3cd91a1909e2f) All that's changed is that this line has moved, which I don't think is necessary.